### PR TITLE
Update notify_python35.cpp

### PR DESCRIPTION
**Pull request by pintomax**

### CODE REVIEW

File: notify_python35.cpp [hunk 1]
```diff
@@ -136,20 +136,12 @@ bool NotifyPython35::configure()
...
 
-#if 0
 	// 2) Import Python script
-	// check first method name (substring of scriptname) has name SCRIPT_NAME
-	if (filterMethod.compare(SCRIPT_NAME) != 0)
+	// check first method name is empty:
+	// disable delivery, cleanup and return true
+	// This allows reconfiguration
+        if (filterMethod.empty())
 	{
```
Comment (AmandeepArora) [inline notify_python35.cpp]: May be add a log message here indicating plugin has been disabled.

Comment (pintomax) [reply notify_python35.cpp]: There is already a comment in an early stage about scripName/filterMethod.

The code here silently allows re-configuration if the plugin is setup without a script: i.e the GUI step by step notification delivery setup

**Comment by mshariq-nerd** [comment]
@pintomax  issues on **Buster**
1. Followed steps mention on  the ticket. All goes well 
2. Then disable notification service and re-enable it. 
Error in syslog and notification service move into unresponsive state.

`Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: Signal 11 (Segmentation fault) trapped:
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (0) 0      0x43324 handler(int) + 36---------
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (1) 1   0x76932120 __default_sa_restorer + 0---------
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (2) 2      0x76800 ./foglamp.services.notification() [0x76800]---------
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (3) 3      0x79d90 NotificationQueue::evalRule(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, AssetData, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, AssetData> > >&, NotificationRule*) + 800---------
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (4) 4      0x7d364 NotificationQueue::processAllDataBuffers(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) + 1944---------
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (5) 5      0x7d540 NotificationQueue::process() + 300---------
Aug  9 11:09:10 raspberrypi FogLAMP Notifications[2174]: FATAL: (6) 6   0x76b9f9b0 /usr/lib/arm-linux-gnueabihf/libstdc++.so.6(+0x9d9b0) [0x76b9f9b0]---------
Aug  9 11:09:15 raspberrypi FogLAMP Storage[2014]: ERROR: filterPayload: exception Connection refused sending reading data to interested party http://127.0.0.1:33351/notification/reading/asset/sinusoid
Aug  9 11:09:20 raspberrypi FogLAMP Storage[2014]: ERROR: filterPayload: exception Connection refused sending reading data to interested party http://127.0.0.1:33351/notification/reading/asset/sinusoid
Aug  9 11:09:22 raspberrypi FogLAMP[1998] INFO: scheduler: foglamp.services.core.scheduler.scheduler: Process started: Schedule 'stats collection' process 'stats collector' task 6b02e138-db45-4efd-9b52-1500c90901aa pid 2383, 3 running tasks#012['tasks/statistics', '--port=44981', '--address=127.0.0.1', '--name=stats collection']`

**Comment by pintomax** [comment]
@mshariq-nerd There is  new JIRA for Buster: FOGL-3120

**Pull request merged**